src/service/resources-impl.js

```diff
@@ -624,6 +624,39 @@ export class Resources {
   }
 
   /**
+   * Requires the layout of the specified element or top-level sub-elements
```

Comment:
Yes. We currently do not have an option to do recursive filtering well enough. But such a use case is also somewhat rare in animations. I'll follow up on this and see if we can find a way to do this right for all cases.